ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -0,0 +1,411 @@
+import {Component} from '@angular/core';
+import {Button} from 'app/components/buttons';
+import {FadeBox} from 'app/components/containers';
+import {FlexColumn, FlexRow} from 'app/components/flex';
+import {SemiBoldHeader} from 'app/components/headers';
+import {TextArea} from 'app/components/inputs';
+import {BulletAlignedUnorderedList} from 'app/components/lists';
+import {Modal, ModalBody, ModalFooter, ModalTitle} from 'app/components/modals';
+import {TooltipTrigger} from 'app/components/popups';
+import {Scroll} from 'app/icons/scroll';
+import {TextInputWithLabel} from 'app/pages/login/account-creation/common';
+import {institutionApi} from 'app/services/swagger-fetch-clients';
+import colors from 'app/styles/colors';
+import {reactStyles, ReactWrapperBase, UrlParamsProps, withUrlParams} from 'app/utils';
+import {navigate} from 'app/utils/navigation';
+import {DuaType, Institution, OrganizationType} from 'generated/fetch';
+import * as fp from 'lodash/fp';
+import {Dropdown} from 'primereact/dropdown';
+import * as React from 'react';
+import * as validate from 'validate.js';
+
+const styles = reactStyles({
+  label: {
+    fontSize: '14px',
+    fontWeight: 600,
+    letterSpacing: 0,
+    lineHeight: '22px',
+    color: colors.primary,
+    marginTop: '2rem',
+    marginBottom: '0.3rem'
+  }
+});
+
+interface InstitutionEditState {
+  apiErrorMsg: string;
+  isAddInstitution: boolean;
+  institution: Institution;
+  invalidEmailAddress: boolean;
+  invalidEmailAddressMsg: string;
+  invalidEmailDomain: boolean;
+  invalidEmailDomainsMsg: string;
+  showOtherInstitution: boolean;
```

Comment:
This makes me think it means "show other institutions" so we should rename it to something which makes it clear it means "show the other-text field"